modules/comb.h

```diff
@@ -0,0 +1,67 @@
+// #Comb
+// Comb filter module
+// ## Credits
+// *Original author(s) :
+// *Ported from soundpipe by Ben Sergentanis, May 2020
+
+#pragma once
+#ifndef DSY_COMB_H
+#define DSY_COMB_H
+
+#include <stdint.h>
+#ifdef __cplusplus
+
+#include <math.h>
+
+namespace daisysp
+{
+
+class Comb
+{
+    public:
+    Comb() {}
+    ~Comb() {}
+  
+    // ### Init
+    //
+    // Initializes the Comb module.
+    //
+    // sample_rate - The sample rate of the audio engine being run. 
+    // 
+    // ~~~~
+    void Init (float sample_rate, float* buff, size_t size);
+    // ~~~~
+
+    // ### Process
+    //
+    // 
+    // ~~~~
+        float Process(float in);
+    // ~~~~
+
+    // #Setters
+    //
+    // ### SetFreq
+    //
+    // 
+    // ~~~~
+	inline void SetFreq(float looptime);
```

Comment:
This can't be called, 'inline' and then be defined else where. Otherwise you get `undefined reference to 'daisysp::Comb::SetFreq(float)'

removing inline fixes this.